pyaerocom/helpers.py

```diff
@@ -836,6 +841,12 @@ def resample_timeseries(ts, freq, how=None, min_num_obs=None):
         p = int(how.split("percentile")[0])
         how = lambda x: np.nanpercentile(x, p)  # noqa: E731
 
+    if how == "error":
+        if not isinstance(ts, pd.Series):
+            raise NotImplementedError("Not yet implemented.")
+
+        how = resample_errors
```

Comment:
This looks wrong. `resample_errors` for a `mean` needs a `1/n` factor, not only the sqrt of the square-sums. 
mean = $\frac{\sum_{i=1}^n ts_i}{n}$
mean-error = $\frac{\sqrt{\sum_{i=1}^n ts_i^2}}{n}$

Error for percentiles: should be derived from both the distribution of values and their errors. @lewisblake : some statistics please...

But error-resampling depends on the data-resampling method and should be somehow connected.

Reply:
Resolved by a conversation :)